chroma-manager/tests/feature/cli/features/steps/standard_cli.py

```diff
@@ -91,5 +91,5 @@ def step(context, message):
         ok_(message not in "".join(context.stdout.readlines()))
     except AssertionError:
         context.stdout.seek(0)
-        print context.stdout.readlines()
+        print(context.stdout.readlines())
```

Comment:
if you are going to subsume another patch in this one it should be clearly documented, should be rebasing on patches you need functionality from